sky/cli.py

```diff
@@ -4632,8 +4632,10 @@ def serve_status(all: bool, endpoint: bool, service_names: List[str]):
               default=False,
               required=False,
               help='Skip confirmation prompt.')
+@click.option('--replica-id', '-r', default=None, type=int, help='')
 # pylint: disable=redefined-builtin
-def serve_down(service_names: List[str], all: bool, purge: bool, yes: bool):
+def serve_down(service_names: List[str], all: bool, purge: bool, yes: bool,
```

Comment:
To discuss: should we introduce a new CLI?
Pros: no confusion; could let user specify multiple replica ids in a single CLI
Cons: adds complecity to our system
cc @Michaelvll

Reply:
From discussion offline, we agreed that we will keep everything in a single CLI